apps/parsley/src/components/styles/sectionHeader.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from "@leafygreen-ui/emotion";
+import { palette } from "@leafygreen-ui/palette";
+import { size } from "constants/tokens";
+
+const { gray } = palette;
+
```

Comment:
it looks like the sections are not rendering properly:

<img width="434" alt="Screenshot 2024-08-23 at 10 19 56 AM" src="https://github.com/user-attachments/assets/51cf0147-80f3-42bb-9ab1-adf6630df17a">